src/Storage/Sha1FileHashingService.php

```diff
@@ -150,8 +178,8 @@ private function getCache()
      */
     private function buildCacheKey($fileID, $fs)
     {
-        $fs = $this->normaliseFilesystem($fs);
-        $fsID = spl_object_hash($fs);
+        $fsID = $this->getFilesystemKey($fs);
```

Comment:
I couldn't rely `spl_object_hash($fs)` any more to get a key for my filesystem, because that would only return a consistent value for the current request ... I need to value to FS Key to be consistent from request to request.